### DISCUSSION TITLE
Library of Congress CIP?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#464] [Get Help]
Has anyone here registered their epublication with the Library of Congress? If so, what was the timeline? Their website suggests submitting the request six months prior to publication, which seems like a long time for an epub. If you did register, did you go live before or after?

**Comment by Erin-Cecele** [comment]
Hi @mmpadilla15. I spoke with my colleague and she pointed out that the website says: "If the print date is less than 8 weeks away, please use the LCCN form instead."

**Reply by mmpadilla15** [reply]
@Erin-Cecele Thank you - I'll let our editor know.

**Reply by dimrkh** [reply]
Hi @mmpadilla15, please note that "At this time, books published only in electronic format are not eligible for the CIP Program. Books published simultaneously in print and electronic format are eligible." –– I was just reviewing the CIP terms and found this; thought it might be relevant. Perhaps and LCCN would work if your pub is electronic only?
